src/korean.js

```diff
@@ -93,6 +96,13 @@ class Korean {
     } // end of first else
   } // end of presentWord function
 
+  doPresentContinuous(word) {
+    let wordLength = word.length
+    let conjugate = word.split("").slice(0, wordLength-1)
+    conjugate.push('고','있','어')
+    return conjugate = conjugate.join("")
```

Comment:
Ideally, we should have eslint set up, but before we get that - please put a ; after each line for better style.